### DISCUSSION TITLE
Can you Integrate .asm or Assembly Files to your operating system to add more functionality to it?

### DISCUSSION THREAD

**Discussion by RafCodes-del** [CosmosOS/Cosmos#3049] [General]
Yeah im just asking about this is this possible? or no. because im already trying to code a file system in assembly and i want to integrate it with my operating system.

Edit:  Ive been getting errors with the VFS file system on cosmos.

**Comment by zarlo** [comment]
X# plugs https://cosmosos.github.io/articles/Kernel/Plugs.html#assembly-plug

**Reply by RafCodes-del** [reply]
Thanks. im gonna try this!
